src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body{
+  margin: 0;
+  padding: 0;
+}
```

Comment:
According to the checklist, you should avoid using tag names for styling (except `html` and `body`). In this case, you are using the `body` tag which is acceptable. However, you are using the `*` selector to set styles for all elements which can impact performance. It's better to set styles only for elements that require them.